internal/version/version.go

```diff
@@ -82,11 +92,14 @@ type Version struct {
 	URL     string `json:"url"`
 }
 
-// SemVerRegex is the regular expression used to parse a semantic version.
-// https://github.com/Masterminds/semver/blob/master/LICENSE.txt
-const SemVerRegex string = `v?([0-9]+)\.([0-9]+)\.([0-9]+)`
+// Rollout represents the state of a version rollout.
+type Rollout struct {
+	Seed   rolloutSeed `json:"seed"`
+	Cursor int32       `json:"cursor"`
+	Target Version     `json:"target"`
```

Comment:
Why do we need this `Target`? I thought the rollout is for the `Suggested` version.

Reply:
"TargetVersion" is a field of rollout in the design doc. I don't know what the expected relationship is, if any, between the rollout and the suggested version.

Reply:
Hmm... yes, we have target version in the design doc..

@littleskunk Do we really need "target" version in addition to the "suggested" version. If yes, what's the difference?